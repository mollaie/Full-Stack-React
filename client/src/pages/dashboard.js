import Layout from "../components/layout";
import Home from "./home";

const Dashboard = () => {
  return (
    <Layout>
      <Home />
    </Layout>
  );
};

export default Dashboard;
