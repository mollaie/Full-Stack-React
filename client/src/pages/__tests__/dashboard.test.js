import React from "react";
import renderer from "react-test-renderer";
import dashboard from "../dashboard";

describe("dashboard test", () => {
  it("dashboard should match snapshot", () => {
    const component = renderer.create(<dashboard />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
