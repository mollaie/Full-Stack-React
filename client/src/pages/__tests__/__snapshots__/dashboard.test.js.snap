// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard test dashboard should match snapshot 1`] = `<dashboard />`;
